lib/decompilerl.ex

```diff
@@ -1,11 +1,98 @@
 defmodule Decompilerl do
 
+  def summarise(module, device \\ :stdio) do
+    obtain_beam(module)
+    |> make_summary
+    |> format_summary
+    |> write_to(device)
+  end
+
   def decompile(module, device \\ :stdio) do
     obtain_beam(module)
     |> do_decompile
     |> write_to(device)
   end
 
+  defp format_summary(map) do
+    %{:file       => file,
+      :module     => module,
+      :behaviours => behaviours,
+      :exports    => exports,
+      :functions  => functions} = map
+    lines = [
+      "File         : " <> file,
+      "Module       : " <> module,
+      format(behaviours, "Behaviours   : "),
```

Comment:
Would you consider using `String.pad_trailing/3` for all the formatting touch ups?